### DISCUSSION TITLE
An important parameter is missing from the API doc on editing a post

### DISCUSSION THREAD

**Discussion by GreenFootballs** [mastodon/mastodon#25651] [General]
I'm working on a function to edit a Mastodon post, and could not find a way in the API documentation to  edit the ALT text attributes of images.

That section of the docs is [here.](https://docs.joinmastodon.org/methods/statuses/#edit)

I finally used the DevTools network tab to watch the API calls as I edited an image's description and discovered a parameter being passed that is not in the doc: `media_attributes`, an array of objects that look like this:

```
"media_attributes": [
    {
        "id": "110566601931157298",
        "description": "White priest: \"Without us you'd still be worshipping the sun!\"\n\nNative Americans: \"Dude... the sun is real\"",
        "focus": "0.00,0.00"
    }
]
```

The API doc does have a parameter for `media_ids`, but not `media-attributes`. And without using it there is no way to edit ALT text, a very common usage because people often forget to add it the first time they post.

**Comment by trwnh** [comment]
hi! please file this as an issue against mastodon/documentation: https://github.com/mastodon/documentation

(the reason it's missing is due to a more recent change, btw -- it was added in 4.1.0 and #20878 while the linked section of the docs is only updated for 4.0.0)

EDIT: see https://github.com/mastodon/documentation/issues/1211 ; it is noted in there
